vulnerable_people_form/tier_lookup/lacode_tier_loookup.py

```diff
@@ -0,0 +1,66 @@
+import bios
+import subprocess
+from shutil import copyfile, rmtree
+from _datetime import datetime
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+LOCAL_RESTRICTIONS_FILE = "local-restrictions.yaml"
+PATH_TO_RESTRICTIONS_FILE = f"vulnerable_people_form/tier_lookup/{LOCAL_RESTRICTIONS_FILE}"
+COLLECTIONS_GIT_REPO = "https://github.com/alphagov/collections.git"
+DEFAULT_NATIONAL_ALERT_LEVEL = 3
+
+
+class LacodeTierLookup:
+    """
+    fetches the tier status for supplied LA Code
+    """
+
+    def __init__(self,
+                 load_tier_data: bool = False,
+                 tier_file_path: str = None):
+
+        self.tier_file_path = tier_file_path
+        if load_tier_data:
+            self.fetch_tier_data_file()
+
+    def fetch_tier_data_file(self):
+        args = ["git", "clone", "--depth=1", COLLECTIONS_GIT_REPO]
+        logger.info(f"Cloning Git repo - {COLLECTIONS_GIT_REPO}")
+        res = subprocess.Popen(args, stdout=subprocess.PIPE)
+        output, _error = res.communicate()
+        logger.info(f"Finished Cloning Git repo - {COLLECTIONS_GIT_REPO}")
+
+        if not _error:
+            copyfile(f"collections/lib/local_restrictions/{LOCAL_RESTRICTIONS_FILE}", PATH_TO_RESTRICTIONS_FILE)
+            rmtree("collections")
+        else:
+            logger.error(f"Unable to fetch new local restrictions file, defaulting to existing file: {_error}")
+
+    def lacode_to_tier(self, lacode: str, date: datetime = datetime.now()):
+        if self.tier_file_path is None:
+            self.tier_file_path = LOCAL_RESTRICTIONS_FILE
+
+        my_dict = bios.read(self.tier_file_path)
```

Comment:
Can we use `PyYAML` here? I don't understand what `bios` is and why we'd need it.

Reply:
Replaced bios with PyYAML